### ISSUE TITLE
Thold Upgrade - database import

### ISSUE THREAD

**Issue by ntt1985** [Cacti/plugin_thold#105]
Hi, I'd like to know if it could be possible to import the database in a brand new version of cacti with one of the latest version of the plugin? More precisely I have an old installation:
Cacti 0.8.6h with thold 0.3.4 running on a Centos 5.11 32bit system
and I'd like to import the database (of the plugin) in a brand new installation with
cacti 1.1.16 and thold 1.0.3 running on a Centos 7.3 64bit box

Basically the database structure in the old version (0.3.4) can be checked here : http://pastebin.centos.org/172641/

If I can import the db, how can I need to proceed? Should I extract the content of the plugin archive in the plugins/ folder and then run the extras/upgrade.php script before install the plugin in cacti?
Should I import all tables?
Please, give me some good advice.

Thank you

**Comment by cigamit** [comment]
I have not spent any time on it.  The official position is that it is not supported, but some people have managed to properly upgrade the system.
